supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -27,3 +27,30 @@ Provide the source language if it is supported by the Red Hat Customer Portal to
 == Long code examples in procedures
 
 All code blocks (regardless of length) must be necessary, accurate, and helpful. Code blocks must be as copy-and-paste friendly as possible, with the exception of variables and callouts. The length of the block is irrelevant, within reason, if the code block follows these guidelines.
+
+[[command-line-commands-that-require-elevated-privileges]]
+== Command-line commands that require elevated privileges
```

Comment:
How about:

_Elevated privileges CLI commands_

Full CLI term is defined in following text.

Reply:
If you're looking to shorten the heading, then I would suggest one of the following:
_Commands with elevated privileges_
_Command that require elevated privileges_

I don't like the sound of **_Elevated privileges_ CLI commands**, and I think that CLI here is not necessary.

Reply:
Good suggestion.

I could +1 "Commands with elevated privileges"

Reply:
Changed to`Commands with elevated privileges`